src/welcome-to-compute-@-edge.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Welcome to Compute@Edge</title>
+    <link
+      rel="icon"
+      href="https://developer.fastly.com/favicon-32x32.png"
+      type="image/png"
+    />
+  </head>
+  <body>
+    <iframe
+      src="https://developer.fastly.com/compute-welcome"
+      style="
+        border: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+      "
+    ></iframe>
+  </body>
+</html>
```

Comment:
The benefit of using the iframed content is that we can update it in one place without having to update each starter kit individually.